templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,11 @@
+# Configure authentication methods
+
+{!admin-only.md!}
+
+1. Open up the organization settings using [this guide](/templates/zerver/help/change-your-organization-settings.md).
+
+2. Select the Authentication Methods tab on the left sidebar.
+
+3. Enable or disable the authentication methods you want.
+
+    ![authentication methods](/static/images/help/authentication-methods-tab.png)
```

Comment:
Where's the image? Or does it exist already?

Reply:
Oh oops, forgot to commit that :) Will do.